package.json

```diff
@@ -41,6 +41,9 @@
     "husky": "4.3.6"
   },
   "scripts": {
-    "build": "ncc build --source-map && ./patch-preset-loader.sh"
+    "build": "npm run clean && npm run build:action && npm run build:presets",
```

Comment:
👍 

Non-blocking

I am seeing an error locally for `codemirror` when calling `npm run build`.  Note: only failing for this one preset:

<img width="961" alt="Screenshot 2023-11-15 at 11 31 31 AM" src="https://github.com/Clearcover/conventional-changelog-action/assets/33120/2e2f0a5d-4f65-4d53-83b0-201b91fd0933">

Reply:
Fixed in commit https://github.com/Clearcover/conventional-changelog-action/pull/21/commits/668c2415fe854be7933a4ff69fb0ca93033a8d7f.